File: 1-Iniciante/ex1064.js
// entrada
let input = require('fs').readFileSync('dev/stdin', 'utf-8'); //adicionar uma barra antes do 'dev' para funcionar
let numbers = input.split('\n').map(n => parseFloat(n));

// processamento
let cont_positivos = soma = 0;
numbers.map(n => {
    if(n > -1){
        cont_positivos++;
        soma += n;
    }
})

// saida
console.log(`${cont_positivos} valores positivos`);
console.log((soma / cont_positivos).toFixed(1));

File: 1-Iniciante/ex1064.py
numeros = []
positivos = 0
media = 0 

#entrada & processamento
for i in range(0, 6):
    numeros.append(float(input()))
    if numeros[i] >= 0:
        positivos += 1
        media += numeros[i]

media = media / positivos

#saída
print(f'{positivos} valores positivos')
print(f'{media:.1f}')
